app/component/itinerary/navigator/NaviBoardingInfo.js

```diff
@@ -12,9 +12,16 @@ const NaviBoardingInfo = ({
   headsign,
   translationValues,
   withExpandIcon,
+  compact,
 }) => {
   return (
-    <div className={cx('boarding', { 'with-icon': withExpandIcon })}>
+    <div
+      className={cx(
+        { boarding: !compact },
+        { 'compact-boarding': compact },
+        { 'with-icon': withExpandIcon },
+      )}
```

Comment:
Could be simplified into
```suggestion
      className={cx({ 
      boarding: !compact,
      'compact-boarding': compact,
      'with-icon': withExpandIcon
      })}
```